ztools/x509/x509.go

```diff
@@ -948,10 +958,38 @@ func parseSANExtension(value []byte) (dnsNames, emailAddresses []string, ipAddre
 			return
 		}
 		switch v.Tag {
+		case 0:
+			var oName pkix.OtherName
+			_, err = asn1.UnmarshalWithParams(v.FullBytes, &oName, "tag:0")
+			if err == nil{
```

Comment:
nit: this is conventionally written as `if err != nil { return } // rest of your code`. 

You can also add the assignment to the if statement in golang:
```
if _, err = asn1.UnmarshalWithParams(v.FullBytes, &oName, "tag:0"); err != nil {
     return
}
```